frontend/public/components/RBAC/role.jsx

```diff
@@ -238,6 +244,29 @@ export const BindingsForRolePage = (props) => {
   );
 };
 
+const getBreadcrumbPath = (plural) => {
+  const lastNamespace = sessionStorage.getItem(LAST_NAMESPACE_NAME_LOCAL_STORAGE_KEY);
+  return lastNamespace === ALL_NAMESPACES_KEY || lastNamespace === null
```

Comment:
I realize you can't use `getBreadcrumbPath` from breadcrumbs.ts because the path is different, but we should factor out the common logic for determining the correct namespace to avoid duplicating it. The knowledge of where it is in `sessionStorage` and how the breadcrumb namespace is determined should only be in one place.

Reply:
Actually, can you use `getBreadcrumbPath` from breadcrumbs.ts and pass a `customPlural` here?

Reply:
Ha!  "great minds...", yes, but I also had to add create/mock a `match.params.ns` since cluster resources don't have it set:
```
const getBreadcrumbs = (modelRef, kindObj, match) => {
  const model = modelFor(modelRef);
  return [
    {
      name: i18next.t(model.labelPluralKey) || model.labelPlural,
      path: getBreadcrumbPath(
        { params: { ns: sessionStorage.getItem(LAST_NAMESPACE_NAME_LOCAL_STORAGE_KEY) } },
        model.plural,
      ),
```

Reply:
Added WIP label, noticed latest changes not quite right.